selenium_tests/course_info/page_objects/course_info_search_page_object.py

```diff
@@ -81,3 +87,16 @@ def select_course(self, cid=None, title=None):
         # loading the results can take a long time, so explicitly wait longer
         WebDriverWait(self._driver, 30).until(lambda s: s.find_element(
             *CoursePeoplePageLocators.ADD_PEOPLE_BUTTON).is_displayed())
+
+    def get_text(self, course_code_text):
```

Comment:
This is very vague -- get what text? from where? Also note that it isn't getting the _course code_ text -- it's just matching a TD with the text you send it when you call the method.